agent/pkg/dagent/utils/docker.go

```diff
@@ -405,9 +406,12 @@ func WithImportContainer(dc *containerbuilder.DockerContainerBuilder, importConf
 			mountList []mount.Mount,
 			logger *io.StringWriter) error {
 			if initError := spawnInitContainer(client, ctx, containerName, mountList, importConfig, dog, cfg); initError != nil {
-				log.Printf("Failed to spawn init container: %v", initError)
+				errStr := fmt.Sprintf("Failed to spawn init container: %v", initError)
+				log.Println(errStr)
```

Comment:
log.Println?

Reply:
Yes, for info, if something goes ham, letting that be visible.